### PR TITLE
some minor fixes

### PR DESCRIPTION
**Pull request by juangirini**

### CODE REVIEW

File: src/Timelock.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ function Timelock() {
     <section id="timelock" className="overflow-hidden relative min-h-screen bg-left bg-no-repeat bg-cover bg-black flex flex-col flex-col-reverse md:flex-row">
       <div className="container flex items-center justify-end">
         <div className="textContainer flex flex-col justify-center items-end gap-10 relative">
-          <h1 className="text-4xl xl:text-6xl font-bold text-right">Timelock Encryption</h1>
+          <h2 className="text-4xl xl:text-6xl font-bold text-right">Timelock Encryption</h2>
           <p className="text-right">The Ideal Network enables timelock encryption, allowing you to effortlessly send messages into the future and unlock them right from your browser!</p>
-          <a href="https://etf.idealabs.network/docs/intro" className="cta" target="_blank" rel="noreferrer">Encryption to the Future</a>
+          <a href="https://etf.idealabs.network/docs/intro" className="cta" target="_blank" rel="noreferrer">Encryption To the Future</a>
```
Comment (driemworks) [inline src/Timelock.js]: Should we also capitalize 'The'? Either "to the" or "To The" would be fine

Comment (carloskiron) [reply src/Timelock.js]: agreed

Comment (driemworks) [reply src/Timelock.js]: The general rule is to not capitalize three letter (or less) words unless they're prinicipal words, so articles like 'the' and connecting words like 'to' should be lowercase.

**Comment by driemworks** [comment]
LGTM

**Pull request merged**